apps/spruce/src/pages/image/tabs/EventLogTab/EventLogTab.test.tsx

```diff
@@ -380,6 +380,48 @@ describe("image event log page", async () => {
       expect(rows).toHaveLength(3);
     });
   });
+
+  it("supports global filter for existing entries", async () => {
```

Comment:
could we add a test for the functionality of the global filter versus the table filter? I think the global filter probably takes precedence, could we test for that?

Reply:
bump on this: could we write an additional test `global filter takes precedence over table filters`

and what would happen is that you could apply a name search within the table to show results, then apply a global filter on top of it which would change the displayed results